workspaces/orchestrator/app-config.yaml

```diff
@@ -81,8 +81,10 @@ auth:
 # permission:
```

Comment:
Dear reviewer, this is intentionally commented-out by default :-)
We usually do not need this feature for day-to-day development.